dynatrace-aws-logs.sh

```diff
@@ -139,6 +167,7 @@ arguments:
 
   aws cloudformation deploy --stack "$STACK_NAME" --template-file "$TEMPLATE_FILE" --capabilities CAPABILITY_IAM \
     --parameter-overrides DynatraceEnvironmentUrl="$TARGET_URL" DynatraceApiKey="$TARGET_API_TOKEN" VerifySSLTargetActiveGate="$REQUIRE_VALID_CERTIFICATE" \
+    UseExistingActiveGate="$USE_EXISTING_ACTIVE_GATE" TenantIdNewAGonly="$TENANT_ID" DynatracePaasToken="$TARGET_PAAS_TOKEN" \
```

Comment:
I'd just call it TenantId, it's bit noisy to read in cloudFormation later on the way it is

Reply:
True that, changing